### PR TITLE
Update kafka, module executions times and jaeger properties

### PR DESCRIPTION
**Pull request by jedla97**
### Summary

Updating kafka images to latest versions

Remove `quarkus.skip.build`  from example/management pom to speed up native builds

Add `quarkus.skip.build` to example/funqy-knative-events as this is only executed on openshift and have fixed build time properties

Update jaeger properties as Quarkus migrate from OpenTracing to OpenTelemetry (see [big-bang-change-from-opentracing-to-opentelemetry](https://quarkus.io/version/main/guides/telemetry-opentracing-to-otel-tutorial#big-bang-change-from-opentracing-to-opentelemetry))

Please check the relevant options

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Dependency update
- [x] Refactoring
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] This change requires a documentation update
- [x] This change requires execution against OCP (use `run tests` phrase in comment)

### Checklist:
- [ ] Example scenarios has been updated / added
- [ ] Methods and classes used in PR scenarios are meaningful
- [x] Commits are well encapsulated and follow [the best practices](https://cbea.ms/git-commit/)

### CODE REVIEW

**Comment by jedla97** [comment]
run tests

File: quarkus-test-service-kafka/src/main/java/io/quarkus/test/services/containers/model/KafkaRegistry.java [hunk 1]
```diff
@@ -1,8 +1,8 @@
 package io.quarkus.test.services.containers.model;
 
 public enum KafkaRegistry {
-    CONFLUENT("confluentinc/cp-schema-registry", "7.5.3", "/", 8081),
-    APICURIO("quay.io/apicurio/apicurio-registry-mem", "2.5.8.Final", "/apis", 8080);
+    CONFLUENT("confluentinc/cp-schema-registry", "7.6.1", "/", 8081),
```
Comment (michalvavrik) [inline quarkus-test-service-kafka/src/main/java/io/quarkus/test/services/containers/model/KafkaRegistry.java:4]: Let's read https://quarkus.io/guides/kafka-schema-registry-avro#confluent first before you bump it. Does it work in native? It didn't for me, please recheck.

Comment (michalvavrik) [reply quarkus-test-service-kafka/src/main/java/io/quarkus/test/services/containers/model/KafkaRegistry.java:4]: CI seems it pass, so maybe it's docs issue and this change is fine.

**Review by michalvavrik — APPROVED** [review]
Please make sure you have green OCP CI before you merge this. I'll be off so just go ahead. Changes LGTM.

**Comment by jedla97** [comment]
run tests

**Pull request merged**